src/widgets/list.rs

```diff
@@ -23,6 +23,16 @@ impl Default for ListState {
 }
 
 impl ListState {
+    /// Returns the offset.
```

Comment:
Do we need these comments ?

Asking because the rest of the code doesn't seem to use comments like these.

Reply:
Even if there aren't many doc comments as of now, why not start?

Reply:
I personally prefer comments only when strictly needed, the code you wrote looks self explanatory to me, so we don't need to maintain code + comments.

But I'll leave this to the maintainer to review 👌